packages/client/src/index.ts

```diff
@@ -27,16 +27,19 @@ export type {
   OsdkActionParameters,
   OsdkObject,
   PageResult,
+  Result,
   WhereClause,
 } from "@osdk/client.api";
+
+export { isOk } from "@osdk/client.api";
 export { PalantirApiError } from "@osdk/shared.net.errors";
 
 export type { Client } from "./Client.js";
 export { createClient } from "./createClient.js";
 export { createPlatformClient } from "./createPlatformClient.js";
 
 export { createAttachmentFromRid } from "./createAttachmentFromRid.js";
+export { createAttachmentUpload } from "./object/AttachmentUpload.js";
```

Comment:
Need this if someone wants to upload a blob to an action. The `AttachmentUpload` type should also accept the File object from FileApi but we need to provide this helper for now because that doesn't come standard yet